web/src/resources/TeamResources.ts

```diff
@@ -25,3 +37,59 @@ export async function updateTeamForRobot(
   assertHttpCode(response.status, 200);
   return response.data?.name;
 }
+
+
+export async function updateTeamRoleForNamespace(
+  namespace: string,
+  teamName: string,
+  teamRole: string,
+) {
+  const updateTeamUrl = `/api/v1/organization/${namespace}/team/${teamName}`;
+  const payload = {name: teamName, role: teamRole};
+  const response: AxiosResponse = await axios.put(updateTeamUrl, payload);
+  assertHttpCode(response.status, 200);
+  return response.data?.name;
+}
+
+
+export async function fetchTeamsForNamespace(
+  org: string,
+  signal?: AbortSignal,
+) {
+  const teamsForOrgUrl = `/api/v1/organization/${org}`;
+  const teamsResponse = await axios.get(teamsForOrgUrl, {signal});
+  assertHttpCode(teamsResponse.status, 200);
+  return teamsResponse.data?.teams;
+}
+
+export async function deleteTeamForOrg(orgName: string, teamName: string) {
+  try {
+    await axios.delete(`/api/v1/organization/${orgName}/team/${teamName}`);
+  } catch (err) {
+    console.error('Unable to delete team', err);
+  }
+}
+
+
+export async function bulkDeleteTeams(orgName: string, teams: ITeams[]) {
+  const responses = await Promise.allSettled(
+    teams.map((team) => deleteTeamForOrg(orgName, team.name)),
+  );
+
+  // Aggregate failed responses
+  const errResponses = responses.filter(
```

Comment:
Can we use [throwIfError](https://github.com/quay/quay/blob/master/web/src/resources/ErrorHandling.ts#L31-L46) here?